browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -4457,7 +4368,6 @@ L.CanvasTileLayer = L.Layer.extend({
 		if (this._docType === 'spreadsheet') {
 			map.on('zoomend', this._onCellCursorShift, this);
 		}
-		map.on('zoomend', L.bind(this.eachView, this, this._viewCursors, this._onUpdateViewCursor, this, false));
```

Comment:
this handler is removed, but do we have some replacement which happens after zoom change?

Reply:
> this handler is removed, but do we have some replacement which happens after zoom change?

Thanks for checking. Yes we a call under "_onZoomEnd" function.